Emrald_Site/EditForms/VariableEditor.js

```diff
@@ -136,7 +136,7 @@ function validateValue() {
     }
   }
   else if (scope.doubleTypeSelected) {
-    if (typeof (scope.data.value) === "number" && scope.data.value.toString().match(/^-?[0-9]*.[0-9]*$/)) {
+    if (scope.data.value.toString().match(/^-?[0-9.]+([eE][-+]?[0-9.]+)?$/)) {
```

Comment:
This would allow for something like `1.....1....2....3...e....3.....3..4` to be valid.  You need to take the decimal out of those brackets and add a second optional digit group.  Probably something more like this:
```
^-?((\d+(\.?\d+)?)([eE][-+]?\d+(\.?\d+)?)?)$
```

Reply:
Same for the EventEditor